### ISSUE TITLE
Notifications on Challenges' comment's replies

### ISSUE THREAD

**Issue by Kimberley-Heath** [emfoundation/ce100-app#1331]
Person A posts a challenge & Person B leaves a comment. Person A gets a notification of this. Person A then replies to Person B's comment but Person B is not notified... This creates a stop in the flow of conversation. 

It is not an urgent one for now as users are not leaving many comments but should be considered as engagement on cello increases...

@asimonw

**Comment by asimonw** [comment]
@Kimberley-Heath Agreed that it would be a nice improvement once we see more people commenting.

**Comment by asimonw** [comment]
This should be tied into the wider notifications discussion going on at the moment.
